packages/web/src/non-real-time-vad.ts

```diff
@@ -88,12 +88,13 @@ export class NonRealTimeVAD {
     let end = 0
     let frameIndex = 0
 
-    let messageContainer: FrameProcessorEvent[] = []
-
     for await (const frame of resampler.stream(inputAudio)) {
+      const messageContainer: FrameProcessorEvent[] = []
```

Comment:
This is the key change, moving inside the loop to not allow many events to accumulate and contaminate eachother.